src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -715,7 +715,7 @@ public function isAbstract()
     /**
      * Sets a configurator to call after the service is fully initialized.
      *
-     * @param callable $callable A PHP callable
+     * @param callable|Definition $callable A PHP callable or a service definition
```

Comment:
technically, it is not a callable. It is the definition of a callable (for object callables, it would be ``array(new Reference(...), 'configureThings')``

and it cannot be a Definition object itself. it can be ``array(new Definition(...), 'configureThings')``. the configurator will be a method of a service, not the service itself